nigthOwlTests/PersonTesting.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using nightOwl;
+using nightOwl.Components;
+
+namespace nigthOwlTests
+{
+    [TestClass]
+    public class PersonTesting
+    {
+        Person person1 = new Person(1, "Jonas", "1952-09-25", "2018-04-11");
+        Person person2 = new Person(1, "Petras", "1945-03-21", "2017-02-25");
+        Person person3 = new Person(1, "Petras", "1945-03-21", "2017-02-25");
+
+        [TestMethod]
+        public void ToStringValueIsAName()
+        {
```

Comment:
This test doesn't really test any entity or business logic, maybe only the constructor. You shouldn't test your constructors